.changeset/little-steaks-rule.md

```diff
@@ -0,0 +1,39 @@
+---
+'astro': patch
+---
+
+**BREAKING CHANGE to the experimental SVG Component API only**
+
+- The `title` prop has been removed until we can settle on the correct DX and a11y tradeoffs
+  ```diff
+  - <Logo title="My Company Logo" />
+  + <Logo aria-label="My Company Logo" />
+  ```
+- Sprite mode has been removed until we can find the correct implementation without the footguns of the previous approach.
+  - The `mode` prop has been removed due to the removal of Sprite mode.
+    ```diff
+    - <Logo mode="inline" />
+    + <Logo />
+    ```
+  - The `mode` config option has been removed due to the removal of Sprite mode.
+    ```diff
+    import { defineConfig } from 'astro'
+
+    export default defineConfig({
+    -  experimental: {
+    -    svg: {
+    -      mode: 'sprite'
+    -    },
+    -  }
+    });
+    ```
```

Comment:
``````suggestion
```diff
- <Logo mode="inline" />
+ <Logo />
```

```diff
import { defineConfig } from 'astro'

export default defineConfig({
    experimental: {
  -    svg: {
  -      mode: 'sprite'
  -    },
  -  }
  +  svg: true
});
```
``````
Check the formatting on this, but I think it's sufficient to explain above, then just show the two code samples without nested bulleted lists. Also, they shouldn't delete the experimental config *entirely*, right? It should now just say `true`?

Reply:
Yes that's correct, the option is still under `experimental`.